readme.md

```diff
@@ -43,12 +43,18 @@ $ create-dmg --help
 
 ## DMG
 
-The DMG requires macOS 10.11 or later and has the filename `App Name 0.0.0.dmg`, for example `Lungo 1.0.0.dmg`.
+The DMG detects the minimum runtime of the app, and uses ULFO (macOS 10.11 or later) or UDZO as appropriate. The resulting image has the filename `App Name 0.0.0.dmg`, for example `Lungo 1.0.0.dmg`.
 
 It will try to code sign the DMG, but the DMG is still created and fine even if the code signing fails, for example if you don't have a developer certificate.
 
 <img src="screenshot-dmg.png" width="772">
 
+### Software license
+
+If `license.txt`, `license.rtf`, or `sla.r` (raw SLAResources file) are present in the same folder as the app, they will be added as a software agreement when opening the image. The image will not be mounted unless the user indicates agreement with the license.
+
+`/usr/bin/rez` (Xcode Developer Tools) must be installed.
```

Comment:
```suggestion
`/usr/bin/rez` (Xcode Command Line Tools) must be installed.
```

And linkify it to https://developer.apple.com/download/more/

Reply:
Done